codespell_lib/data/dictionary.txt

```diff
@@ -2701,6 +2701,8 @@ arrangts->arrangements
 arraning->arranging
 arranment->arrangement
 arranments->arrangements
+arrant->arrangement
+arrants->arrangements
```

Comment:
Reason for the checks failure, because `arrant` is a valid word. Do you mind removing `arrant` ?
See https://www.oxfordlearnersdictionaries.com/definition/english/arrant?q=arrant

While `arrants` is not a valid word.